package.json

```diff
@@ -25,14 +25,16 @@
     "istanbul": "*",
     "jasmine": "^2.5.3",
     "jasmine-core": "^2.5.2",
+    "json-loader": "^0.5.4",
     "karma": "^1.4.1",
     "karma-jasmine": "^1.1.0",
     "karma-phantomjs-launcher": "^1.0.2",
     "mocha": "*",
     "repl.history": "*",
     "semver": "*",
     "typescript": "2.0.8",
-    "uglify-js": "2.x"
+    "uglify-js": "2.x",
+    "webpack": "^1.14.0"
```

Comment:
I'm actually hoping we won't need webpack at all. This is a temporary work-around until the issue can be fixed on react-native's side. I used webpack 1 since I had tested with that before webpack 2 was released, and the fields I'm using don't exist in webpack 2 anymore.

If for whatever reason the react native issue can't be fixed, I'll update this to 2+